config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -165,3 +165,61 @@ periodics:
           # this is mostly for building kubernetes
           memory: 14Gi
           cpu: 7
+- interval: 1h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-two-steps-upgrade-n-minus-1
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: two-steps-upgrade-test-n-minus-1
+    description: Uses kind to run e2e tests from the n-1 kubernetes release against a latest kubernetes master components w/ --emulated-version=n-1 set.
+    # TODO(#34269) update owners in experiment/compatibility-versions
+    testgrid-alert-email: compatibility-version-test@google.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+    workdir: true
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:v20250513-98d205aae3-master
+      imagePullPolicy: Always  # pull latest image for canary testing
```

Comment:
this isn't a canary job and shouldn't set `imagePullPolicy: Always`, it's using a stable image with an immutable tag.